src/dotnet-interactive-vscode-ads/src/versionSpecificFunctions.ts

```diff
@@ -3,6 +3,7 @@
 
 import * as path from 'path';
 import * as vscode from 'vscode';
+import * as azdata from 'azdata';
```

Comment:
Is this file only included in the ADS version of the extension or something?

Reply:
Yes, every extension has a file like this. The common code references this file, so they all need to have the same function signatures.